DesktopEdge/Views/Screens/IdentityDetails.xaml

```diff
@@ -1,16 +1,16 @@
 ﻿<UserControl x:Class="ZitiDesktopEdge.IdentityDetails"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:local="clr-namespace:ZitiDesktopEdge"
-             mc:Ignorable="d" 
+             mc:Ignorable="d"
              x:Name="IdentityArea"
              d:DesignHeight="500" d:DesignWidth="350" IsVisibleChanged="VisibilityChanged">
     <Grid>
         <!-- Border Arrow pointing to TaskBar -->
         <Canvas>
-            <Rectangle Name="Arrow" HorizontalAlignment="Center" Height="20" Fill="White" Width="20" Canvas.Left="185" ClipToBounds="True" Canvas.Bottom="-10">
+            <Rectangle Name="Arrow" HorizontalAlignment="Center" Height="20" Fill="White" Width="20" ClipToBounds="True">
```

Comment:
Removes arrow alignment static assignment.  Refer to ...
https://github.com/openziti/desktop-edge-win/pull/756/commits/9e595faa0483521a7d65dce790db29a1b798c255#r1847092571